src/containers/nav/desktop/CurrentlyPlaying.module.css

```diff
@@ -32,6 +31,39 @@
   cursor: pointer;
 }
 
+.bottomRightContainer {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 50px;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.visualizerIconContainer {
+  padding: 5px;
+  background-color: white;
```

Comment:
`background-color: var(--static-white)`

is probably what you want here. this will stay #FFF always, but also use our theme. question for julian probably about whether we should flip this to a dark theme in dark mode